Test/Unit/Model/Api/DebugTest.php

```diff
@@ -200,7 +213,11 @@ public function debug_successful()
 						'name' => 'plugin3',
 						'version' => '3.0.0'
 					]
-				]
+				],
+                'logInfo' => [
```

Comment:
Seems like you don't have the latest code. We now use the snake case for field names. Or let's change it to logs for simplicity. Check this PR for reference: https://github.com/BoltApp/bolt-magento2/pull/621

Reply:
Ah yes, this was originally written friday. Will update.